pkg/controller/registry/reconciler/configmap_test.go

```diff
@@ -342,6 +350,24 @@ func TestConfigMapRegistryReconciler(t *testing.T) {
 				},
 			},
 		},
+		{
+			testName: "ExistingRegistry/BadNetworkPolicy",
+			in: in{
+				cluster: cluster{
+					k8sObjs: append(setLabel(objectsForCatalogSource(t, validCatalogSource), &networkingv1.NetworkPolicy{}, CatalogSourceLabelKey, "wrongValue"), validConfigMap),
+				},
+				catsrc: validCatalogSource,
+			},
+			out: out{
+				status: &v1alpha1.RegistryServiceStatus{
+					CreatedAt:        now(),
+					Protocol:         "grpc",
+					ServiceName:      "cool-catalog",
+					ServiceNamespace: testNamespace,
+					Port:             "50051",
```

Comment:
Nit: We use this port number across multiple places for Catalog-related services and such. Would love to have a const for this to synchronize them if we ever decide to change it. However, it is not a big deal. Just a note here.

Reply:
Noted, and agreed. That change might be better left as its own PR though.